README.md

```diff
@@ -9,16 +9,54 @@ https://zookeeper.ibinx.com/master/
 
 ### Requirements
 
-* PHP 5.6 or later with MySQL PDO driver
-* MySQL/MariaDB
-
+* Apache 2.4
+* MySQL/MariaDB 5.6+
+* PHP 7.2 & PHP 7.2-mysql (thread safe version required)
 
 ### Getting Started
 
-1. Clone the repository;
+
+The PHP setup can be installed either by manually installing its constituent
+components or by installing and integrated 'XAMPP' framework. The procedure
+using the latter is as follows:
+
+  - cd <SRC_DIR>
+  - git clone https://github.com/RocketMan/zookeeper.git
+  - git checkout -b kzsu
+  - Install XAMPP, eg www.apachefriends.org and run XAMPP Control Panel
```

Comment:
XAMPP is one way to do it, but many environments support reasonably intuitive/turn-key integration of PHP, Apache, and MySQL without this.  Debian for example.  Some platforms don't even have XAMPP.  Bottom line, exact steps to setup the environment is beyond the scope of zookeeper, but we could mention that one must install PHP, Apache, and MySQL through whatever platform-specific means one wants, or use an integration framework such as XAMPP.  I'll leave it to you to decide what to say, but we cannot leave it with XAMPP as a required step.

Reply:
XAMPP is not a required step. text clearly states that it is *one* option.